src/core/electrostatics_magnetostatics/fft.cpp

```diff
@@ -283,12 +283,12 @@ void pack_block_permute1(double const *const in, double *const out,
   auto const s_in_offset = element * (dim[2] * (dim[1] - size[1]));
   /* offset for mid changing indices of output grid */
   auto const m_out_offset = (element * size[0]) - element;
-  /* linear index of in grid, linear index of out grid */
+  /* linear index of in grid */
   int li_in = element * (start[2] + dim[2] * (start[1] + dim[1] * start[0]));
-  int li_out = 0;
 
   for (int s = 0; s < size[0]; s++) { /* fast changing out */
-    li_out = element * s;
+    /* linear index of out grid */
```

Comment:
same here

Reply:
that's the name of the argument `double const *const out`